package.json

```diff
@@ -5,7 +5,7 @@
     "@types/debug": "4.1.5",
     "@types/express": "4.17.11",
     "@types/node": "14.14.31",
-    "@types/socket.io": "2.1.4",
+    "@types/socket.io": "^3.0.2",
```

Comment:
the package should have its own types now so let's remove

Reply:
removed